config/initializers/differ.rb

```diff
@@ -0,0 +1,6 @@
+require_dependency 'differ/differ'
+require_dependency 'differ/simple_diff'
+
+if ENV['DIFFER_SOURCE']
+  Differ.register(:source, Differ::SimpleDiff.new(ENV['DIFFER_SOURCE']))
```

Comment:
Would it be too clever to allow arbitrary many differs with arbitrary names to be configured through `ENV` by picking up all vars with names that match `DIFFER_*`? Or will new differs likely need something more complex that `SimpleDiff`? Offhand I'm not sure what they would need so the approach I'm floating seems _doable_ -- but maybe less than _clean_.